src/codegen/codegen_c_visitor.cpp

```diff
@@ -986,17 +986,17 @@ std::vector<SymbolType> CodegenCVisitor::get_shadow_variables() {
 /****************************************************************************************/
 
 std::string CodegenCVisitor::get_parameter_str(const ParamVector& params) {
-    std::stringstream param_ss;
+    std::string param{};
```

Comment:
I am curious. Why does this function needs to change to using string as well?

Reply:
it does not have to. But this is the last one using stringstream in this file, all other methods use fmt.

Reply:
I see. Thanks for the answer!